custom/icds_reports/migrations/0104_agg_ls_monthly_ls_name.py

```diff
@@ -17,4 +17,3 @@ class Migration(migrations.Migration):
     operations = [
         migrator.get_migration('update_tables44.sql'),
     ]
-    operations.extend(get_view_migrations())
```

Comment:
I needed to prevent the building of other views building because I need to rebuild the agg_awc_monthly view with new columns.